Sources/Public/iOS/AnimatedControl.swift

```diff
@@ -132,6 +132,11 @@ open class AnimatedControl: UIControl {
     updateForState()
   }
 
+  public override var accessibilityTraits: UIAccessibilityTraits {
+    set { customAccessibilityTraits = newValue }
+    get { customAccessibilityTraits.union(.button).union(super.accessibilityTraits) }
+  }
```

Comment:
I wonder if we could avoid the stored `customAccessibilityTraits` property with an implementation like this:

```suggestion
  public override var accessibilityTraits: UIAccessibilityTraits {
    set { super.accessibilityTraits = newValue }
    get { super.accessibilityTraits.union(.button) }
  }
```

Reply:
or do we specifically need to not modify `super.accessibilityTraits` to get the UIControl provided states like when `isEnabled = false` or `isSelected = true`?

Reply:
Good question, confirming now.

Reply:
Your simplification works and maintains the UIControl provided states! Will update.

Reply:
Nice! Thanks